react-ui/js/index.js

```diff
@@ -102,7 +122,6 @@ const functions = {
     app.webapp.updateOption('optUseHttps', val);
     if (app.webapp.started) {
       app.webapp.stop();
-      app.webapp.start();
```

Comment:
maybe it could auto start if they already generated the certificates. but yeah maybe this is fine

Reply:
I don't think that is the problem. I have tried changing to many different methods of restarting and it simply isn't possible. The reason it broke was it was trying to start the server while it was stopping (javascript synchronous functions). The only really way is to use like a delay function in javascript (and I know none)